modules/pekko-http/build.sbt

```
@@ -0,0 +1,15 @@
+import Dependencies.Version._
+
+crossScalaVersions := Seq(scala212, scala213)
+
+libraryDependencies ++= Seq(
+  "org.apache.pekko" %% "pekko-stream" % "1.0.2" % "provided",
```

I think only pekko-actor is required here for `ActorRefFactory`.

```suggestion
  "org.apache.pekko" %% "pekko-actor" % "1.0.2" % "provided",
```

---

That seems to work. Changed in f022109